src/include/daos_srv/bio_types.h

```diff
@@ -0,0 +1,130 @@
+/**
```

Comment:
Making this separated header Is only for exporting "bio_dev_state"? If so, I suggest to rename the "bio_dev_state" to some common name and move it to a common header file shared by control plane and data plane.

Reply:
I couldn't find such a place, could you please suggest one?